docs/generate_reference.py

```diff
@@ -1,39 +1,47 @@
 """Generate reference pages for the documentation."""
 
 from pathlib import Path
-from typing import TYPE_CHECKING
+from typing import TYPE_CHECKING, List, Dict, Union, Any, TypedDict
 
 if TYPE_CHECKING:
-    import mkdocs.config.defaults  # pragma: no cover
-
+    pass  # pragma: no cover
```

Comment:
I believe this block can now be deleted :)

Reply:
Yes indeed

Reply:
Indeed, I hadn't seen it, I remember that during the pylint I had an error in docs/generate_reference.py, I let copilot correct it without checking behind.